mc.js

```diff
@@ -207,8 +207,8 @@ mix(Mc.prototype,{
       get value(){
         return buf;
       },
-      set value(val,mode){
-        buf = mode=='append'?buf+val:(mode=='prepend'?val+buf:val);
+      set value(val){
```

Comment:
Yeah, somewhere between `0.9` and `0.12` nodejs started to care about ES syntax :)